### PR TITLE
CUSTOMERS-18: Support for acquisition & reconstruction with wedge

### PR DESCRIPTION
**Pull request by PiotrKarwat**
Support for SSTA acquisition and reconstruction when using wedge interface.

### CODE REVIEW

**Comment by pjarosik** [comment]
@PiotrKarwat resolve conflicts first.

File: api/matlab/examples/Us4R_control.m [hunk 1]
```diff
@@ -2,30 +2,31 @@
...
-txFrequency = 6e6;
+txFrequency = 2.5e6;
 samplingFrequency = 65e6;
 fsDivider = 1;
 
 [filtB,filtA] = butter(2,[0.5 1.5]*txFrequency/(samplingFrequency/fsDivider/2),'bandpass');
 
 %% Initialize the system, sequence, and reconstruction
-us	= Us4R('probeName',   'SL1543', ...
+us	= Us4R('probeName',   '2.25L32', ...
```
Comment (pjarosik) [inline api/matlab/examples/Us4R_control.m:12]: This script should stay as before (i.e. without the interface, e.g. using SL1543 probe).

**Review by pjarosik — CHANGES_REQUESTED** [review]
Please rollback the changes in Us4R_control.m script.

File: api/matlab/examples/Us4R_control.m (hunk 1 above)
Comment (PiotrKarwat) [reply api/matlab/examples/Us4R_control.m]: OK

**Review by pjarosik — APPROVED** [review]

**Pull request merged**